style.css

```diff
@@ -339,9 +358,7 @@ canvas#curve {
 	user-select: none;
 }
 
-section {
-	margin: 2em 0;
-}
+section {}
```

Comment:
I don't usually write my CSS this way, so I'm not entirely sure what would be desirable in this case

Reply:
This is fine